praw/models/reddit/live.py

```diff
@@ -570,6 +597,62 @@ def update(
         self.thread._reset_attributes(*data.keys())
 
 
+class LiveThreadStream:
+    """Provides a :class:`.LiveThread` stream.
+
+    Usually used via:
+
+    .. code-block:: python
+
+       for live_update in reddit.live("ta535s1hq2je").stream.updates():
+           print(live_update.body)
+
+    """
+
+    def __init__(self, live_thread: LiveThread):
+        """Create a LiveThreadStream instance.
+
+        :param live_thread: The live thread associated with the stream.
+        """
+        self.live_thread = live_thread
+
+    def updates(
+        self, **stream_options: Dict[str, Any]
+    ) -> Iterator["LiveUpdate"]:
+        """Yield new updates to the live thread as they become available.
+
+        :param skip_existing: Set to ``True`` to only fetch items created
+            after the stream (default: ``False``).
+
+        As with :meth:`.LiveThread.updates()`, updates are yielded as
+        :class:`.LiveUpdate`.
+
+        Updates are yielded oldest first. Up to 100 historical updates will
+        initially be returned.
+
+        Keyword arguments are passed to :func:`.stream_generator`.
+
+        For example, to retrieve all new updates made to the
+        ``"ta535s1hq2je"`` live thread, try:
+
+        .. code-block:: python
+
+           for live_update in reddit.live("ta535s1hq2je").stream.updates():
+               print(live_update.body)
```

Comment:
Could you make this indented 4 spaces instead of 3? I plan on standardizing this at some point. 
```suggestion
            for live_update in reddit.live("ta535s1hq2je").stream.updates():
                print(live_update.body)
```